src/webhook/mutation/pod_mutator/init_container.go

```diff
@@ -27,22 +28,39 @@ func createInstallInitContainerBase(webhookImage, clusterID string, pod *corev1.
 			{Name: config.K8sNamespaceEnv, ValueFrom: kubeobjects.NewEnvVarSourceForField("metadata.namespace")},
 			{Name: config.K8sNodeNameEnv, ValueFrom: kubeobjects.NewEnvVarSourceForField("spec.nodeName")},
 		},
-		SecurityContext: copyUserContainerSecurityContext(pod),
+		SecurityContext: securityContextForInitContainer(pod),
 		Resources:       *dynakube.InitResources(),
 	}
 }
 
-func copyUserContainerSecurityContext(pod *corev1.Pod) *corev1.SecurityContext {
-	var securityContext *corev1.SecurityContext
-	if len(pod.Spec.Containers) == 0 {
-		return securityContext
-	}
+func securityContextForInitContainer(pod *corev1.Pod) *corev1.SecurityContext {
+	var securityContext = new(corev1.SecurityContext)
```

Comment:
```suggestion
	var securityContext = &corev1.SecurityContext{
	  RunAsNonRoot = address.Of(true),
	  ...
	}
```

I'd just create the limited security context in the beginning since we always need it

Reply:
done